CHANGELOG.md

```diff
@@ -148,3 +148,7 @@ All notable changes to this project will be documented in this file. The format
 ### Changed:
```

Comment:
Shouldn't we update the `Added` section with the following features:
- Elastic IP support for AWS.
- EVH support for OpenShift
- NSX-T cloud support for VLAN and overlay based segments.

Reply:
Pick up the change log from all the epics that we have fixed in 1.5.1.

Reply:
@sudswasavi : Addressed.